tests/test_routes.py

```diff
@@ -36,7 +36,7 @@ def test_create_application_is_successful(client):
         "account_id": "userb",
         "fund_id": "fund-b",
         "round_id": "summer",
-        "language": "en",
+        "language": "null",
```

Comment:
Think this might need to be `None` test the null database case?

Reply:
changed to None 👍🏾